src/PrestaShopBundle/Bridge/AdminController/Action/AbstractAction.php

```diff
@@ -34,7 +34,7 @@
  * This class is a base class for action, which are actions the user can do.
  * For example, add a new entity, or update it.
  */
-class Action implements ActionInterface
+abstract class AbstractAction implements ActionInterface
```

Comment:
Why set is as abstract? It has no abstract methods

Reply:
Mainly because it shouldn't be used directly, so far it's always been extend This way it allows checking the type of the action in the addAction method and this way the actions are automatically added in their proper container